vlog4j-core/src/main/java/org/semanticweb/vlog4j/core/model/implementation/Serializer.java

```diff
@@ -384,9 +386,15 @@ private static String addQuotes(final String string) {
 		return QUOTE + string + QUOTE;
 	}
 
-
 	private static String addAngleBrackets(final String string) {
 		return LESS_THAN + string + MORE_THAN;
 	}
 
+	public static String getFactOutput(Predicate predicate, List<Term> terms) {
```

Comment:
I don't know if the word "output" makes sense in the method name. Why not call it getString or getFactString ?